src/nvim/api/vim.c

```diff
@@ -1562,3 +1563,20 @@ Object nvim_get_proc(Integer pid, Error *err)
 #endif
   return rvobj;
 }
+
+/// Give the info of the state of the Caps/Num/Scroll lock.
+///
+/// This info will be in form of the dictionary, having
+/// `CapsLock`, `NumLock` and `ScrollLock` as the keys of dictionary
```

Comment:
This now sounds as if all keys are always present.

```C
/// Get information about keyboard locks state
///
/// @return dictionary with up to three keys, depending on which checks are
///         implemented: `CapsLock`, `NumLock` and `ScrollLock`. Corresponding
///         values are booleans, true if relevant lock is active and false
///         otherwise. Dictionary may contain more keys in the future.
```

Reply:
OK, I edit it